pyproject.toml

```diff
@@ -48,12 +50,18 @@ ignore = [
     "ANN204", # Missing return type annotation for special method
     "E501",   # Line too long
 ]
+"*.py" = [
+    "E501",   # Line too long
+    "FBT001", # Boolean positional argument in function definition (should be keyword-only)
+    "FBT002", # Boolean default positional argument in function definition (should be keyword-only)
+]
```

Comment:
These should be scoped only to the `fsspec.py` file

Reply:
I pushed a quick fix to this: [`897beb0` (#198)](https://github.com/developmentseed/obstore/pull/198/commits/897beb07b84d5fda85776551015ae4ff09408a37)